a10_octavia/controller/worker/flows/a10_load_balancer_flows.py

```diff
@@ -706,6 +706,11 @@ def get_delete_rack_vthunder_load_balancer_flow(self, lb, cascade, vthunder_conf
         delete_LB_flow.add(nat_pool_tasks.NatPoolDelete(
             requires=(constants.LOADBALANCER, a10constants.VTHUNDER,
                       a10constants.LB_COUNT_FLAVOR, constants.FLAVOR_DATA)))
+        delete_LB_flow.add(a10_database_tasks.CountLBThunderPartition(
```

Comment:
Is it better if we move these 2 task after DisableLBAmphoraeHealthMonitoring task below?
So, put disable HM for octavia and ACOS tasks together